website/src/components/ComparisonTable.tsx

```diff
@@ -410,14 +411,15 @@ const taskRunnerRows: Comparison[] = [
 		feature: 'Can generate run reports',
 		support: {
 			moon: SUPPORTED,
+			turborepo: SUPPORTED,
 		},
 	},
 	{
 		feature: 'Continuous integration (CI) support',
 		support: {
 			moon: SUPPORTED,
 			nx: PARTIALLY_SUPPORTED,
-			turborepo: PARTIALLY_SUPPORTED,
+			turborepo: SUPPORTED,
```

Comment:
https://turbo.build/repo/docs/ci

Reply:
This isn't correct. Turbo doesn't have a built-in CI method similar to `moon ci`.